### PR TITLE
`new post` CSS text-wrap: pretty

### PR DESCRIPTION
**Pull request by argyleink**
ToDo:
- [x] get final review
- [x] update date

### CODE REVIEW

File: site/en/blog/css-text-wrap-pretty/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  - argyle
+tags:
+  - css
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/spOTYANOG4RylwyIQyRH.png
+thumb: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/mY6BvPXwqyfz7Jn70jwG.png
+alt: >
+  A tall building's windows are shown wrapping in a harmonious curve.
+---
+
+Set to release in Chrome 117 is a new text wrapping feature—`text-wrap: pretty`
```
Comment (rachelandrew) [inline site/en/blog/css-text-wrap-pretty/index.md:19]:
```suggestion
From Chrome 117 you can use a new text wrapping feature—`text-wrap: pretty`
```

File: site/en/blog/css-text-wrap-pretty/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+<figure>
+  {% Img
+    src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/wfqjbgG80KgZryht7iiK.svg",
+    alt="A paragraph is shown with a widow at the beginning and an orphan at the end, compared to the same paragraph without the orphans or widows.",
+    width="800",
+    height="450"
+  %}
+
+  <figcaption>
+    Image sourced from <a href="https://fonts.google.com/knowledge/glossary/widows_orphans">Google Fonts - Widows & Orphans</a>
```
Comment (rachelandrew) [inline site/en/blog/css-text-wrap-pretty/index.md:63]:
```suggestion
    Image sourced from <a href="https://fonts.google.com/knowledge/glossary/widows_orphans">Google Fonts—Widows & Orphans</a>
```

File: site/en/blog/css-text-wrap-pretty/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+<figure>
+  {% Img
+    src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/Snt1oo5WBdJeXVE8Wcrc.png",
+    alt="A comparison of a paragraph with orphans and one with no orphans, each with a badge of bad or good.",
+    width="540",
+    height="250"
+  %}
+
+  <figcaption>
+    Image sourced from <a href="https://uxmovement.com/content/why-you-should-remove-orphans-from-your-body-text/">Why you should remove orphans from your body text</a>
```
Comment (rachelandrew) [inline site/en/blog/css-text-wrap-pretty/index.md:91]:
```suggestion
    Image sourced from <a href="https://uxmovement.com/content/why-you-should-remove-orphans-from-your-body-text/">Why you should remove orphans from your body text.</a>
```

File: site/en/blog/css-text-wrap-pretty/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+harmonious text block. I personally use `balance` for headlines and `pretty` for
+paragraphs.
+
+If you're interested in the details of the algorithm used to determine the
+optimal number of lines, or performance considerations, [here's a link to the
+design document created by the
+engineer](https://docs.google.com/document/d/1jJFD8nAUuiUX6ArFZQqQo8yTsvg8IuAq7oFrNQxPeqI/edit#heading=h.cqq9czoal00g)
+behind the feature, [Koji Ishii](/authors/kojiishi/).
+
+If you have other line breaking improvements or suggestions, we'd love to hear
```
Comment (rachelandrew) [inline site/en/blog/css-text-wrap-pretty/index.md:106]: Add a link to the tracker.

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Just a couple of things to fix.

File: site/en/blog/css-text-wrap-pretty/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+them! File an issue in the Chromium bug tracker with the details, examples of
```
Comment (argyleink) [inline site/en/blog/css-text-wrap-pretty/index.md:107]:
```suggestion
them! File an issue in the [Chromium bug tracker](https://bugs.chromium.org/p/chromium/issues/entry) with the details, examples of
```

File: site/en/blog/css-text-wrap-pretty/index.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
+---
+layout: layouts/blog-post.njk
+title: "CSS text-wrap: pretty"
+description: >
+  Opt-in optimized text wrapping, for beauty over speed.
+subhead: >
+  Opt-in optimized text wrapping, for beauty over speed.
+date: 2023-10-18
```
Comment (argyleink) [inline site/en/blog/css-text-wrap-pretty/index.md:8]:
```suggestion
date: 2023-10-23
```

**Comment by argyleink** [comment]
> Just a couple of things to fix.

ty! fixes are in 👍🏻

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**